hazelcast-sql/src/test/java/com/hazelcast/sql/SqlErrorTest.java

```diff
@@ -72,57 +67,22 @@ public void testMemberLeave() {
         instance1 = newHazelcastInstance(false);
         instance2 = newHazelcastInstance(true);
 
-        populate(instance1);
-
-        // Block query execution on a remote member
-        BlockingExec.Blocker blocker = new BlockingExec.Blocker();
-
-        setExecHook(instance2, exec -> {
-            if (exec instanceof MapScanExec) {
-                return new BlockingExec(exec, blocker);
-            }
-
-            return exec;
-        });
+        SqlStatement streamingQuery = new SqlStatement("SELECT * FROM TABLE(GENERATE_STREAM(50000))");
```

Comment:
Should not we start query first, then shutdown a member?

Reply:
We have `Thread.sleep` below.

Reply:
We have a timeout in thread above.

Reply:
`Thread.sleep()` does not guarantee any ordering, we should start the query first then having that shutdown a member.

Reply:
It doesn't guarantee any ordering, correct. But, 
I see no way to shutdown the member after the query start and gain `MemberLeftException`, only with small timeout before shutdown.

Reply:
I've introduce tests for both cases.

Reply:
Would not starting the query in one thread (making sure at least one row is fetched) and shutting down member in another work?

Reply:
Done, thanks for noticing it!